modules/hcp-support-matrix.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * hosted-control-planes/index.adoc
+
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-support-matrix"_{context}"]
+= Support matrix for {hcp}
+
+Because {mce} includes the HyperShift Operator, releases of {hcp} align with releases of {mce-short}. For more information, see "OpenShift Operator Life Cycles".
+
+Any supported {product-title} cluster can be a management cluster. The following table maps {mce-short} versions to the management cluster versions that support them:
+
+.Supported {mce-short} versions for {product-title} management clusters
+[cols="2",options="header"]
+|===
+|Management cluster version |Supported {mce-short} version
```

Comment:
https://access.redhat.com/support/policy/updates/openshift_operators#:~:text=Multi%2DCluster%20Engine we can point here for a more sustainable tracking of the versions as well.

Reply:
Yes the MCE part which is burried under the ACM one.

Reply:
Got it. I wasn't able to add a link directly to that section, but I did add a link to the OCP Operator Update Information Checker: https://access.redhat.com/labs/ocpouic/?operator=multicluster-engine&&upgrade_path=4.14%20to%204.16